web/src/main/webapp/WEB-INF/jsp/managestudy/updateStudyEventSigned.jsp

```diff
@@ -583,14 +583,14 @@
 <div id="subjDiscNoteDiv" class="subjDiscNoteDiv" style="display:none">
```

Comment:
line 163 has the words "Study Event" hardcoded as well.  This should use the property file.